### ISSUE TITLE
wallet service fails to start with default servers

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-sdk#2243]
## The Issue
A user is having trouble connecting with lbryumx2/4, and the sdk does not try the other ones properly. It even says connected successfully, but it never syncs the account. 

Bad: 
```2019-06-17 17:21:06,311 INFO     lbrynet.extras.daemon.Daemon:402: Starting LBRYNet Daemon
2019-06-17 17:21:06,312 INFO     lbrynet.extras.daemon.Daemon:404: Platform: {
  "processor": "x86_64",
  "python_version": "3.7.1",
  "platform": "Linux-4.15.0-51-generic-x86_64-with-debian-stretch-sid",
  "os_release": "4.15.0-51-generic",
  "os_system": "Linux",
  "lbrynet_version": "0.37.4",
  "torba_version": "0.5.4a0",
  "build": "release",
  "distro": {
    "id": "ubuntu",
    "version": "16.04",
    "version_parts": {
      "major": "16",
      "minor": "04",
      "build_number": ""
    },
    "like": "debian",
    "codename": "xenial"
  },
  "desktop": "Unity"
}
2019-06-17 17:21:08,160 INFO     lbrynet.extras.daemon.Daemon:412: lbrynet API listening on TCP 127.0.0.1:5279
2019-06-17 17:21:08,162 INFO     lbrynet.extras.daemon.Daemon:418: lbrynet media server listening on TCP 127.0.0.1:5280
2019-06-17 17:21:09,610 INFO     lbrynet.extras.daemon.Components:78: Loading databases
2019-06-17 17:21:09,611 INFO     torba.client.basedatabase:189: connecting to database: /home/amyr/.local/share/lbry/lbrynet/lbrynet.sqlite
2019-06-17 17:21:09,612 INFO     lbrynet.extras.daemon.Components:566: detecting external ip
2019-06-17 17:21:09,877 INFO     lbrynet.extras.daemon.exchange_rate_manager:212: Starting exchange rate manager
2019-06-17 17:21:09,999 INFO     lbrynet.extras.daemon.Components:500: found upnp gateway: Huawei Technologies Co., Ltd. HG532e
2019-06-17 17:21:10,005 INFO     lbrynet.extras.daemon.Components:511: got external ip from UPnP: 5.202.7.203
2019-06-17 17:21:10,005 INFO     lbrynet.extras.daemon.Components:525: add UPnP port mappings
2019-06-17 17:21:10,028 INFO     lbrynet.extras.daemon.Components:532: set up redirects: {'TCP': 3333, 'UDP': 4444}
2019-06-17 17:21:10,859 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx2.lbry.com:50001
2019-06-17 17:21:11,167 INFO     lbrynet.extras.daemon.Components:194: remote height: 585872, local height: 585872
2019-06-17 17:21:11,757 INFO     lbrynet.extras.daemon.Components:266: Starting torba wallet
2019-06-17 17:21:11,831 INFO     lbrynet.extras.daemon.Components:345: start the dht
2019-06-17 17:21:11,832 INFO     lbrynet.extras.daemon.Components:370: Started the dht
2019-06-17 17:21:11,833 INFO     torba.client.basedatabase:189: connecting to database: /home/amyr/.local/share/lbry/lbryum/lbc_mainnet/blockchain.db
2019-06-17 17:21:11,840 INFO     lbrynet.dht.node:118: DHT node listening on UDP 0.0.0.0:4444
2019-06-17 17:21:12,371 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx4.lbry.com:50001
```

With lbryumx1 in place in config: 
```
2019-06-17 15:27:11,931 INFO     lbrynet.extras.daemon.Daemon:404: Platform: {
  "processor": "x86_64",
  "python_version": "3.7.1",
  "platform": "Linux-4.15.0-51-generic-x86_64-with-debian-stretch-sid",
  "os_release": "4.15.0-51-generic",
  "os_system": "Linux",
  "lbrynet_version": "0.37.4",
  "torba_version": "0.5.4a0",
  "build": "release",
  "distro": {
    "id": "ubuntu",
    "version": "16.04",
    "version_parts": {
      "major": "16",
      "minor": "04",
      "build_number": ""
    },
    "like": "debian",
    "codename": "xenial"
  },
  "desktop": "Unity"
}
2019-06-17 15:27:13,291 INFO     lbrynet.extras.daemon.Daemon:412: lbrynet API listening on TCP 127.0.0.1:5279
2019-06-17 15:27:13,293 INFO     lbrynet.extras.daemon.Daemon:418: lbrynet media server listening on TCP 127.0.0.1:5280
2019-06-17 15:27:14,230 INFO     lbrynet.extras.daemon.exchange_rate_manager:212: Starting exchange rate manager
2019-06-17 15:27:14,230 INFO     lbrynet.extras.daemon.Components:78: Loading databases
2019-06-17 15:27:14,231 INFO     torba.client.basedatabase:189: connecting to database: /home/amyr/.local/share/lbry/lbrynet/lbrynet.sqlite
2019-06-17 15:27:14,232 INFO     lbrynet.extras.daemon.Components:566: detecting external ip
2019-06-17 15:27:14,568 INFO     lbrynet.extras.daemon.Components:500: found upnp gateway: Huawei Technologies Co., Ltd. HG532e
2019-06-17 15:27:14,574 INFO     lbrynet.extras.daemon.Components:511: got external ip from UPnP: 5.202.0.40
2019-06-17 15:27:14,575 INFO     lbrynet.extras.daemon.Components:525: add UPnP port mappings
2019-06-17 15:27:14,612 INFO     lbrynet.extras.daemon.Components:532: set up redirects: {'TCP': 3333, 'UDP': 4444}
2019-06-17 15:27:15,151 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx1.lbry.com:50001
2019-06-17 15:27:15,358 INFO     lbrynet.extras.daemon.Components:194: remote height: 585829, local height: 585829
2019-06-17 15:27:15,741 INFO     lbrynet.extras.daemon.Components:266: Starting torba wallet
2019-06-17 15:27:15,844 INFO     lbrynet.extras.daemon.Components:345: start the dht
2019-06-17 15:27:15,846 INFO     lbrynet.extras.daemon.Components:370: Started the dht
2019-06-17 15:27:15,852 INFO     torba.client.basedatabase:189: connecting to database: /home/amyr/.local/share/lbry/lbryum/lbc_mainnet/blockchain.db
2019-06-17 15:27:15,861 INFO     lbrynet.dht.node:118: DHT node listening on UDP 0.0.0.0:4444
2019-06-17 15:27:16,583 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx1.lbry.com:50001
2019-06-17 15:27:16,584 INFO     torba.client.baseledger:247: Subscribing and updating accounts.
2019-06-17 15:27:19,027 INFO     lbrynet.wallet.ledger:82: Loaded account bFNn5EEeeTALCNkhqLe6tymPuyKudytGEf with 19.99800408 LBC, 91 receiving addresses (gap: 20), 41 change addresses (gap: 6), 1 channels, 0 certificates and 6 claims. 
2019-06-17 15:27:19,027 INFO     lbrynet.extras.daemon.Components:457: start blob server
2019-06-17 15:27:19,028 INFO     lbrynet.extras.daemon.Components:393: Started blob announcer
2019-06-17 15:27:19,028 INFO     lbrynet.extras.daemon.Components:432: Starting the file manager
2019-06-17 15:27:19,030 INFO     lbrynet.blob_exchange.server:124: Blob server listening on TCP 0.0.0.0:3333
2019-06-17 15:27:19,049 INFO     lbrynet.stream.stream_manager:147: Initializing 13 files
2019-06-17 15:27:19,084 INFO     lbrynet.stream.stream_manager:162: Started stream manager with 13 files
2019-06-17 15:27:19,085 INFO     lbrynet.extras.daemon.Components:438: Done setting up file manager
2019-06-17 15:27:21,121 INFO     lbrynet.dht.protocol.routing_table:289: join buckets 1
2019-06-17 15:27:28,812 INFO     lbrynet.dht.node:158: Joined DHT, 15 peers known in 3 buckets
2019-06-17 15:27:51,643 INFO     lbrynet.extras.daemon.Daemon:690: Shutting down lbrynet daemon
2019-06-17 15:27:51,645 INFO     lbrynet.stream.stream_manager:213: finished stopping the stream manager
2019-06-17 15:27:51,646 INFO     lbrynet.blob_exchange.server:134: Stopped blob server
2019-06-17 15:27:51,646 INFO     lbrynet.extras.daemon.Components:397: Stopped blob announcer
2019-06-17 15:27:51,648 INFO     lbrynet.dht.node:111: Stopped DHT node
2019-06-17 15:27:51,651 INFO     lbrynet.extras.daemon.exchange_rate_manager:217: Stopping exchange rate manager
Task exception was never retrieved
future: <Task finished coro=<MarketFeed._update_price() done, defined at lbrynet/extras/daemon/exchange_rate_manager.py:78> exception=JSONDecodeError('Expecting value: line 1 column 1 (char 0)')>
Traceback (most recent call last):
  File "lbrynet/extras/daemon/exchange_rate_manager.py", line 82, in _update_price
  File "lbrynet/extras/daemon/exchange_rate_manager.py", line 108, in _handle_response
  File "json/__init__.py", line 348, in loads
  File "json/decoder.py", line 337, in decode
  File "json/decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```

## System Configuration

 

- LBRY Daemon version:
- LBRY App version:
- LBRY Installation ID:
- Operating system:

**Comment by tzarebczan** [comment]
Here's an example where it fails to connect to x4 and then doesn't try any other servers. I think this is a high priority.

```2019-06-19 05:24:34,201 INFO     lbrynet.extras.daemon.exchange_rate_manager:212: Starting exchange rate manager
2019-06-19 05:24:34,611 ERROR    torba.client.basenetwork:89: Connecting to lbryumx4.lbry.com:50001 raised an exception:
Traceback (most recent call last):
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/client/basenetwork.py", line 81, in __probe
    await client.create_connection(timeout)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/client/basenetwork.py", line 42, in create_connection
    await asyncio.wait_for(connector.create_connection(), timeout=timeout)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/tasks.py", line 416, in wait_for
    return fut.result()
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/rpc/session.py", line 60, in create_connection
    self.session_factory, self.host, self.port, **self.kwargs)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/base_events.py", line 948, in create_connection
    raise exceptions[0]
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/base_events.py", line 935, in create_connection
    await self.sock_connect(sock, address)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/selector_events.py", line 475, in sock_connect
    return await fut
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/selector_events.py", line 505, in _sock_connect_cb
    raise OSError(err, f'Connect call failed {address}')
OSError: [Errno 113] Connect call failed ('46.101.226.157', 50001)
2019-06-19 05:24:34,684 ERROR    torba.client.basenetwork:89: Connecting to lbryumx4.lbry.com:50001 raised an exception:
Traceback (most recent call last):
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/client/basenetwork.py", line 81, in __probe
    await client.create_connection(timeout)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/client/basenetwork.py", line 42, in create_connection
    await asyncio.wait_for(connector.create_connection(), timeout=timeout)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/tasks.py", line 416, in wait_for
    return fut.result()
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/rpc/session.py", line 60, in create_connection
    self.session_factory, self.host, self.port, **self.kwargs)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/base_events.py", line 948, in create_connection
    raise exceptions[0]
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/base_events.py", line 935, in create_connection
    await self.sock_connect(sock, address)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/selector_events.py", line 475, in sock_connect
    return await fut
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/selector_events.py", line 505, in _sock_connect_cb
    raise OSError(err, f'Connect call failed {address}')
OSError: [Errno 113] Connect call failed ('46.101.226.157', 50001)
2019-06-19 05:24:35,098 INFO     lbrynet.extras.daemon.Daemon:815: Get version info: {"processor": "", "python_version": "3.7.1", "platform": "Android 9 (API 28)", "os_release": "4.14.83-15930013", "os_system": "android", "lbrynet_version": "0.37.4", "torba_version": "0.5.4a0", "build": "release"}
2019-06-19 05:24:35,714 ERROR    torba.client.basenetwork:89: Connecting to lbryumx4.lbry.com:50001 raised an exception:
Traceback (most recent call last):
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/client/basenetwork.py", line 81, in __probe
    await client.create_connection(timeout)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/client/basenetwork.py", line 42, in create_connection
    await asyncio.wait_for(connector.create_connection(), timeout=timeout)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/tasks.py", line 416, in wait_for
    return fut.result()
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/site-packages/torba/rpc/session.py", line 60, in create_connection
    self.session_factory, self.host, self.port, **self.kwargs)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/base_events.py", line 948, in create_connection
    raise exceptions[0]
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/base_events.py", line 935, in create_connection
    await self.sock_connect(sock, address)
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/selector_events.py", line 475, in sock_connect
    return await fut
  File "/data/user/0/io.lbry.browser/files/app/crystax_python/stdlib.zip/asyncio/selector_events.py", line 505, in _sock_connect_cb
    raise OSError(err, f'Connect call failed {address}')
OSError: [Errno 113] Connect call failed ('46.101.226.157', 50001)```

**Comment by tzarebczan** [comment]
Another user ran into something similar, notice no accounts are synced even though it connects: 
```2019-06-20 20:28:32,651 WARNING  lbrynet.extras.daemon.Components:81: db_revision file not found. Creating it
2019-06-20 20:28:32,652 INFO     torba.client.basedatabase:189: connecting to database: /Users/georgedangil/Library/Application Support/LBRY/lbrynet.sqlite
2019-06-20 20:28:32,652 INFO     lbrynet.extras.daemon.exchange_rate_manager:212: Starting exchange rate manager
2019-06-20 20:28:32,671 INFO     lbrynet.extras.daemon.Daemon:815: Get version info: {"processor": "i386", "python_version": "3.7.3", "platform": "Darwin-18.6.0-x86_64-i386-64bit", "os_release": "18.6.0", "os_system": "Darwin", "lbrynet_version": "0.37.5", "torba_version": "0.5.4a0", "build": "release"}
2019-06-20 20:28:33,724 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx2.lbry.com:50001
2019-06-20 20:28:36,207 INFO     lbrynet.extras.daemon.Components:194: remote height: 587380, local height: 0
2019-06-20 20:28:47,715 WARNING  lbrynet.extras.daemon.Components:502: upnp discovery failed: M-SEARCH for 192.168.254.254:1900 timed out
2019-06-20 20:28:47,716 WARNING  lbrynet.extras.daemon.Components:516: unable to get external ip from UPnP, checking lbry.com fallback
2019-06-20 20:28:50,089 ERROR    lbrynet.extras.daemon.Components:581: failed to setup upnp
2019-06-20 20:33:36,572 ERROR    lbrynet.extras.daemon.Components:232: failed to fetch headers from s3: 
2019-06-20 20:33:36,575 INFO     lbrynet.extras.daemon.Components:266: Starting torba wallet
2019-06-20 20:33:36,576 INFO     lbrynet.wallet.manager:153: Wallet at /Users/georgedangil/.lbryum/wallets/default_wallet is empty, generating a default account.
2019-06-20 20:33:36,728 INFO     lbrynet.extras.daemon.Components:345: start the dht
2019-06-20 20:33:36,729 INFO     lbrynet.extras.daemon.Components:370: Started the dht
2019-06-20 20:33:36,730 INFO     torba.client.basedatabase:189: connecting to database: /Users/georgedangil/.lbryum/lbc_mainnet/blockchain.db
2019-06-20 20:33:36,731 INFO     lbrynet.dht.node:118: DHT node listening on UDP 0.0.0.0:4444
2019-06-20 20:33:39,100 INFO     torba.client.basenetwork:108: Successfully connected to SPV wallet server: lbryumx4.lbry.com:50001
2019-06-20 20:33:39,101 INFO     torba.client.baseledger:247: Subscribing and updating accounts.
2019-06-20 20:33:43,410 INFO     lbrynet.dht.protocol.routing_table:289: join buckets 2
2019-06-20 20:33:43,411 INFO     lbrynet.dht.protocol.routing_table:289: join buckets 1
2019-06-20 20:33:51,525 INFO     lbrynet.dht.node:158: Joined DHT, 13 peers known in 3 buckets```

**Comment by tzarebczan** [comment]
Believe this is fixed in 0.38
